ir/memory.cpp

```diff
@@ -16,14 +16,14 @@ namespace {
 // A data structure that represents a byte.
 // A byte is either a pointer byte or a non-pointer byte.
 // Pointer byte's representation:
```

Comment:
uhm, comment with seeing the rest of the file: if we expand to multi-byte, then multiple offset bytes may appear in a memory cell.
I don't think it's possible to mix multiple bids because otherwise a single load would access parts of multiple pointers.
We need, however, ranges of byte offsets when going multi-byte?

Reply:
The range is set to be [offset * byte_size, offset * (byte_size+1) - 1) in this PR.
ex) the offset 0 means bytes 0 ~ 3'th byte of the pointer.